README.md

```diff
@@ -80,7 +80,7 @@ Install a kubernetes cluster
 sealer run kubernetes:v1.19.2 --master 192.168.0.2
 ```
 
-If it is installed on the cloud:
+Install a cluster on alicloud:
```

Comment:
should we mention that we only support alicloud at present, but we are to support other clouds?